src/hooks/validation/models.ts

```diff
@@ -0,0 +1,113 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+interface Specification {
+  isNotValid(): boolean;
+  read(): string;
+  getSpecificationName(): string;
+}
+
+export class SpecificationFile implements Specification {
+  private readonly name: string;
+
+  constructor(name: string) {
+    const localDirectory = process.cwd();
+    this.name = path.join(localDirectory, name);
```

Comment:
```suggestion
    this.name = path.resolve(localDirectory, name);
```

otherwise, when I do `asyncapi validate --context /Users/wookiee/sources/website-asyncapi/asyncapi.yml`
I get error `File: /Users/wookiee/sources/cli/Users/wookiee/sources/website-asyncapi/asyncapi.yml does not exists or is not a file!`

Reply:
didn't notice!! I will fix it!

Reply:
done